x-pack/plugins/rollup/public/crud_app/index.js

```diff
@@ -33,9 +32,7 @@ const renderReact = async (elem) => {
   render(
     <I18nContext>
       <Provider store={rollupJobsStore}>
-        <HashRouter>
-          <App />
-        </HashRouter>
+        <App />
```

Comment:
This changes currently crashes Kibana when accessing the app because the router is needed in the `app.js` `registerRouter()`.

Reply:
Good catch! I fixed this by using an HOC. Maybe this is something we can extract into a root-level `public` dir for global use?